test/unit/definitions/divorce/AuthorisationCaseField.test.js

```diff
@@ -16,25 +16,24 @@ function mergeJsonFilesFor(whatFolder, nonProdStartsWith) {
   const authCaseFieldNonProd = [];
 
   files.forEach(filename => {
-    const filenameRegEx = new RegExp(`${nonProdStartsWith}.*.json`, 'i');
+    const filenameRegEx = new RegExp(`${nonProdStartsWith}.*nonprod.json`, 'i');
     if (filename.match(filenameRegEx)) {
-       const nonProdFile = Object
-          .assign(load(`${whatFolder}/${filename}`), {});
-       authCaseFieldNonProd.push(...nonProdFile);
+      const nonProdFile = Object.assign(load(`${whatFolder}/${filename}`), {});
+      authCaseFieldNonProd.push(...nonProdFile);
     }
   });
 
   return [...authCaseField, ...authCaseFieldNonProd];
 }
 
 describe('AuthorisationCaseField', () => {
-  if (fs.existsSync('definitions/divorce/json/AuthorisationCaseField/')){
-     it('should contain a unique case field ID, case type ID and role (no duplicates) for non prod files', () => {
-        const jsonConfigFiles = mergeJsonFilesFor('definitions/divorce/json/AuthorisationCaseField/', 'AuthorisationCaseField-');
-        const uniqResult = uniqWith(jsonConfigFiles, isDuplicated);
+  if (fs.existsSync('definitions/divorce/json/AuthorisationCaseField/')) {
+    it('should contain a unique case field ID, case type ID and role (no duplicates) for non prod files', () => {
+      const jsonConfigFiles = mergeJsonFilesFor('definitions/divorce/json/AuthorisationCaseField/', 'AuthorisationCaseField-');
+      const uniqResult = uniqWith(jsonConfigFiles, isDuplicated);
 
-        expect(uniqResult).to.eql(jsonConfigFiles);
-     });
+      expect(uniqResult).to.eql(jsonConfigFiles);
+    });
```

Comment:
can you also update this file below:

```
it('should contain a unique case field ID, case type ID and role (no duplicates) for prod file', () => {
    const uniqResult = uniqWith(
      authCaseField,
      (field1, field2) => {
        return field1.CaseTypeID === field2.CaseTypeID
                   && field1.CaseFieldID === field2.CaseFieldID
                   && field1.UserRole === field2.UserRole;
      }
    );
    expect(uniqResult).to.eql(authCaseField);
  });
```

should be:
```
it('should contain a unique case field ID, case type ID and role (no duplicates) for prod file', () => {
    const uniqResult = uniqWith(authCaseField, isDuplicated);

    expect(uniqResult).to.eql(authCaseField);
  });
isDuplicated
```

I know it's not your code, but as you were updating this file, please clean it up a bit.